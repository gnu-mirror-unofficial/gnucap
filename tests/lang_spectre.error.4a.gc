
spectre

// duplicate label error
subckt crash n0 n1 n2 n3 n000 n4
ends

X1 1 2 crash
- list
- op
- end
