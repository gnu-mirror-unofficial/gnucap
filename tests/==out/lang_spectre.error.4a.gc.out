X1 1 2 crash 
   ^ ? 1: no match
// duplicate label error 
subckt crash ()
ends crash

#          
 27.       
